### ISSUE TITLE
Issue when running python manage.py migrate

### ISSUE THREAD

**Issue by AditechGH** [rapidpro/rapidpro#1705]
```
(env) ubuntu@xxxxxxxxxx:~/rapidpro$ python manage.py migrate
/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/utils/request.py:40: UserWarning: python-telegram-bot wasn't properly installed. Please refer to README.rst on how to properly install.
  warnings.warn("python-telegram-bot wasn't properly installed. Please refer to README.rst on "
Traceback (most recent call last):
  File "/home/ubuntu/rapidpro/manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/management/base.py", line 414, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/management/base.py", line 460, in execute
    output = self.handle(*args, **options)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/management/base.py", line 98, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/management/commands/migrate.py", line 91, in handle
    self.check(databases=[database])
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/management/base.py", line 487, in check
    all_issues = checks.run_checks(
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/urls/resolvers.py", line 480, in check
    for pattern in self.url_patterns:
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/utils/functional.py", line 49, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/urls/resolvers.py", line 696, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/utils/functional.py", line 49, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/urls/resolvers.py", line 689, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/rapidpro/temba/urls.py", line 19, in <module>
    re_path(r"^", include("temba.channels.urls")),
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/ubuntu/rapidpro/temba/channels/urls.py", line 13, in <module>
    for ch_type in Channel.get_types():
  File "/home/ubuntu/rapidpro/temba/channels/models.py", line 500, in get_types
    from .types import TYPES
  File "/home/ubuntu/rapidpro/temba/channels/types/__init__.py", line 29, in <module>
    reload_channel_types()
  File "/home/ubuntu/rapidpro/temba/channels/types/__init__.py", line 26, in reload_channel_types
    register_channel_type(import_string(class_name))
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/utils/module_loading.py", line 30, in import_string
    return cached_import(module_path, class_name)
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/django/utils/module_loading.py", line 15, in cached_import
    import_module(module_path)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/ubuntu/rapidpro/temba/channels/types/telegram/__init__.py", line 1, in <module>
    from .type import TelegramType  # noqa
  File "/home/ubuntu/rapidpro/temba/channels/types/telegram/type.py", line 1, in <module>
    import telegram
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/__init__.py", line 101, in <module>
    from .bot import Bot
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/bot.py", line 42, in <module>
    from telegram.utils.request import Request
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/utils/request.py", line 35, in <module>
    import telegram.vendor.ptb_urllib3.urllib3 as urllib3
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/vendor/ptb_urllib3/urllib3/__init__.py", line 7, in <module>
    from .connectionpool import (
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/vendor/ptb_urllib3/urllib3/connectionpool.py", line 30, in <module>
    from .connection import (
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/vendor/ptb_urllib3/urllib3/connection.py", line 39, in <module>
    from .util.ssl_ import (
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/vendor/ptb_urllib3/urllib3/util/__init__.py", line 3, in <module>
    from .connection import is_connection_dropped
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/vendor/ptb_urllib3/urllib3/util/connection.py", line 3, in <module>
    from .wait import wait_for_read
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/vendor/ptb_urllib3/urllib3/util/wait.py", line 1, in <module>
    from .selectors import (
  File "/home/ubuntu/rapidpro/env/lib/python3.10/site-packages/telegram/vendor/ptb_urllib3/urllib3/util/selectors.py", line 11, in <module>
    from collections import namedtuple, Mapping
ImportError: cannot import name 'Mapping' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
```

**Comment by rudigiesler** [comment]
Python 3.10 removed `collections.Mapping` (it's now in `collections.abc.Mapping`). 

It seems like one of RapidPro's dependancies (telegram) is not Python 3.10 compatible.

I'm not sure if there's a newer telegram release available, but for now running it on Python 3.9 (which is what RapidPro's CI runs on) should fix your issue.

**Comment by AditechGH** [comment]
Thanks @rudigiesler 
Running Python 3.9 fixed the problem.
